tests/e2e/.env

```diff
@@ -2,6 +2,10 @@
 # Should be funded with KAVA and have an ERC20 balance
 E2E_KAVA_FUNDED_ACCOUNT_MNEMONIC='tent fitness boat among census primary pipe nose dream glance cave turtle electric fabric jacket shaft easy myself genuine this sibling pulse word unfold'
 
+# E2E_RUN_KVTOOL_NETWORKS is the only currently supported option.
+# It triggers tests to be run against local networks spun up with kvtool.
+E2E_RUN_KVTOOL_NETWORKS=true
```

Comment:
Are you thinking that this will be a `E2E_NODE_NETWORK=kvtool|local|mainnet` type thing when more networks are added or use a config file `E2E_CHAIN_CONFIG_PATH=kvtool.json|mainnet.yaml|testnet.tomlsomeother.combo` or something else?

Reply:
in the branch off this that runs tests on external networks, i just set `E2E_RUN_KVTOOL_NETWORKS` to `false` and then it expects `E2E_KAVA_RPC_URL`, `E2E_KAVA_GRPC_URL`, & `E2E_KAVA_EVM_RPC_URL` env variables to setup the network connection.

i agree it'd be good to extract this into some sort of network configuration file we can add to gitignore to prevent accidental commit of secrets to the repo.